application/src/main/java/org/togetherjava/tjbot/commands/moderation/WarnCommand.java

```diff
@@ -35,25 +28,26 @@
  * added to the database.
  */
 public final class WarnCommand extends SlashCommandAdapter {
```

Comment:
The Javadoc for this is bad. Copy paste the one from `BanCommand` and adjust it instead... That one is much better.